travel-agency-api/src/main/java/cz/muni/fi/pa165/bluebat/dto/ReservationCreateDTO.java

```diff
@@ -22,6 +24,25 @@
     @NotNull
     private PriceDTO priceDTO;
```

Comment:
Asi bych v rezervaci neposílal cenu, ta by se stejně měla znovu přepočítat na serveru, podle toho, co si vybral. O to by se měla postarat service vrstva.